draft-ietf-quic-http.md

```diff
@@ -1192,7 +1191,6 @@ of the associated Element ID fields.
 | 00      | Request stream   | Stream ID                       |
 | 01      | Push stream      | Push ID                         |
 | 10      | Placeholder      | Placeholder ID                  |
-| 11      | Current stream   | Absent                          |
```

Comment:
Since the value of `11` was the only difference between these, we could re-combine them to a single table and simply say that prioritizing Root is invalid.  (Which you already have on line 1213.)

Reply:
That's a nice simplification! Done in 5cca97b.